pkgs/applications/misc/solaar-master/default.nix

```diff
@@ -0,0 +1,42 @@
+{fetchFromGitHub, stdenv, gtk3, python34Packages, gobjectIntrospection}:
+python34Packages.buildPythonApplication rec {
+  name = "solaar-${version}";
+  version = "master";
+  namePrefix = "";
+  src = fetchFromGitHub {
+    owner = "pwr";
+    repo = "Solaar";
+    rev = "master";
+    sha256 = "0zy5vmjzdybnjf0mpp8rny11sc43gmm8172svsm9s51h7x0v83y3";
+  };
+
+  propagatedBuildInputs = [python34Packages.pygobject3 python34Packages.pyudev gobjectIntrospection gtk3];
+  postInstall = ''
+    wrapProgram "$out/bin/solaar" \
+      --prefix PYTHONPATH : "$PYTHONPATH" \
```

Comment:
`--set PYTHONPATH` typically has preference, since otherwise you may break your application if you have some `PYTHONPATH` set.

Reply:
do you mean 
```bash
wrapProgram "$out/bin/solaar" \
    --set PYTHONPATH : "$PYTHONPATH" \
```
?